.env.template

```diff
@@ -1,2 +1,5 @@
 DRIVE_API_URL=
-DRIVE_NEW_API_URL=
\ No newline at end of file
+DRIVE_NEW_API_URL=
+PAYMENTS_API_URL=
+PHOTOS_API_URL=
+REACT_APP_CRYPTO_SECRET=
```

Comment:
remove the "REACT" prefix, this is not a React app and can be confusing, "CRYPTO_SECRET" will work, the "REACT_APP_" prefix is something needed for CRA